controls/raddataservicedatasource/crud.md

```diff
@@ -0,0 +1,56 @@
+---
+title: CRUD
+page_title: CRUD
+description: This article explains how you can perform CRUD operations in RadDataServiceDataSource.
+slug: raddataservicedatasource-crud
+tags: crud
+published: True
+position: 6
+---
+
+# CRUD
+
+You can utilize all CRUD operations by working with the __DataView__ property of the __RadDataServiceDataSource__ control and add, edit or remove entities from the collection which is of type **DataItemCollection**. You can do that either programmatically via the API of the collection or leave the job to __RadGridView__ by binding its __ItemsSource__ to the DataView.
+
+#### __[XAML] Example 1: Bind the DataView collection to RadGridView's ItemsSource__
+{{region raddataservicedatasource-crud_1}}
+    <telerik:RadGridView ItemsSource="{Binding DataView, ElementName=customersDataSource}" IsBusy="{Binding IsBusy, ElementName=customersDataSource}" />
```

Comment:
Can we extend this code snippet a bit by adding also how the DataServiceDataSource is defined. For example:

&lt;Grid>
    &lt;telerik:RadDataServiceDataSource x:Name="customersDataSource" />
    &lt;telerik:RadGridView ..... />
&lt;/Grid>